stores/metadata.go

```diff
@@ -682,11 +719,16 @@ LIMIT ? OFFSET ?`, sqlConcat(s.db, "?", "trimmed"), sqlConcat(s.db, "?", "substr
 }
 
 func (s *SQLStore) Object(ctx context.Context, path string) (object.Object, error) {
-	obj, err := s.object(ctx, path)
-	if err != nil {
-		return object.Object{}, err
-	}
-	return obj.convert()
+	var obj object.Object
+	err := s.db.Transaction(func(tx *gorm.DB) error {
```

Comment:
do we have a unit test for the `Object` method in the store that fetches an object with a partial slab? I don't think I saw one and that might be a good one to add?

Reply:
`TestPartial` slab should do that